### PR TITLE
KCorp Drone Tweaks

### PR DESCRIPTION
**Pull request by TaculoTaculo**
## About The Pull Request
Small tweak to KCorp Drones that spawn rarely in some backstreets rooms, now they flash in an AoE on a cooldown, instead of on every hit. Also makes them non dense. Also removes two redundant lines of code from drones and eye mobs.

## Why It's Good For The Game
KCorp drones were cancerous at best to fight when two caught you at once, now you have some amount of counterplay, but the danger is still there.
Density doesnt really make sense, they are small flying pricks.

## Changelog
:cl:
tweak: tweaks Kcorp drones
balance: Kcorp drone rebalance
code: removes redundant code
/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm [hunk 1]
```diff
@@ -416,25 +416,47 @@
...
+	var/flash_cooldown_time = 10 SECONDS
+	var/flash_range = 5 //Range that the flash hits
+
+/mob/living/simple_animal/hostile/kcorp/drone/proc/Flash()
+	icon_state = "kcorp_drone_angry"
+	can_act = FALSE
+	SLEEP_CHECK_DEATH(2 SECONDS) //Enough to run away, but not easily
+	playsound(src, 'sound/weapons/flash.ogg', 100, TRUE)
+
+	for (var/mob/living/L in viewers(src, null)) //The actual flashing
```
Comment (LanceSmites328) [inline ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm:434]: That's uh, not how you use that proc? `viewers(Depth=world.view,Center=usr)`

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm (hunk 1 above)
Comment (TaculoTaculo) [reply ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm]: Fixed

**Review by Kitsunemitsu — DISMISSED** [review]
Ignore this, it was an error.

**Review by Kitsunemitsu — APPROVED** [review]
Yeah fuck it, looks good to me, and guess what?
Nothing done here could be worse than how it currently is.

**Pull request merged**